satellite/satellitedb/migrate.go

```diff
@@ -785,6 +785,19 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					);`,
 				},
 			},
+			{
+				Description: "Alter value attribution table. Remove bucket_id. Add project_id and bucket_name as primary key",
+				Version:     30,
+				Action: migrate.SQL{
+					`ALTER TABLE value_attributions DROP CONSTRAINT value_attributions_pkey`,
+					`ALTER TABLE value_attributions ADD project_id bytea;`,
+					`UPDATE value_attributions SET project_id=SUBSTRING(bucket_id FROM 1 FOR 16);`,
```

Comment:
I don't think you need logic to move data from one schema to another. We might deploy new table to satellite but I'm pretty sure nobody is using this yet. So just removing column + adding 2 new one should be ok.

Reply:
ah sorry i might be wrong, without this migration can fail because we have one entry in testdata